torch/_inductor/triton_heuristics.py

```diff
@@ -396,7 +397,7 @@ def run(self, *args, grid, stream):
 
         if launcher.config.pre_hook is not None:
             launcher.config.pre_hook(
-                {**zip(self.arg_names, args), **launcher.config.kwargs}
+                {*zip(self.arg_names, args), *launcher.config.kwargs}
```

Comment:
is this not a change in sematnics ?

Reply:
Sorry, I submitted a new commit to fix it.